.github/ISSUE_TEMPLATE/bug-report.md

```diff
@@ -0,0 +1,29 @@
+---
+name: Bug Report
+about: Tell us about a problem with the project's current functionality
+title: "[Bug]"
+labels: bug
+assignees: ''
+
+---
+
+<!--- Provide a general summary of the bug you're experiencing in the Title above -->
+
+
+## Detailed Description
+<!--- Provide a detailed description of the bug you are experiencing -->
+
+
+## To Reproduce
+<!--- Describe the steps that lead to experiencing the bug -->
+
+
+## Your Environment
+<!--- Include as many relevant details about the environment you experienced the bug in, such as the following bullet points -->
+* Version used:
```

Comment:
Version of Parsons used? If so, we should specify. Also possibly give them instructions on how to tell. Does Parsons have a version function?